.storybook/webpack.config.js

```diff
@@ -2,16 +2,16 @@ const path = require('path');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const webpackConf = require('../scripts/webpack.config.js');
 
+const isRelease = process.env.NODE_ENV === 'production';
+const shouldIncludeAllSupportedBrowsers = isRelease || process.env.BROWSERSLIST_ENV === 'production';
```

Comment:
Does this need to be a variable? Can probably just inline it or have something more general like:

```
const isLegacy = process.env.BROWSERSLIST_ENV === 'production';
const isRelease = process.env.NODE_ENV === 'production';

if (isLegacy || isRelease) {
    //...
}
```